crates/biome_js_analyze/src/frameworks/vue/vue_call.rs

```diff
@@ -0,0 +1,60 @@
+use crate::frameworks::is_framework_api_reference;
+use biome_js_semantic::SemanticModel;
+use biome_js_syntax::{AnyJsExpression, JsCallExpression};
+
+/// This function checks if a given call expression is a call to a Vue compiler macro.
+pub fn is_vue_compiler_macro_call(
```

Comment:
nit: This always trips me up because I always interpret this as "is this any macro call" and then get confused why it requires the `macro_name` arg. Perhaps there's a better function name, but I think this doc change is sufficient.

```suggestion
/// Given a Vue compiler macro, checks if the given call expression is a call to that macro.
pub fn is_vue_compiler_macro_call(
```

Reply:
Thanks, applied the change.